src/checkInput.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = (input) => (
+  new Set([...input]).size !== 4
+  || input.length !== 4
+  || [...input].some(d => isNaN(+d))
```

Comment:
would a simple regex do the job here?
`/^\d{4}$/.test(input)`

Reply:
Nice. This would cover the last 2 cases definitely but we would still need the Set to check for repeated numbers

Reply:
true @lefteris-L 
just combine them in a right way )